docs/src/stories/components/Layout/Stack.stories.jsx

```diff
@@ -0,0 +1,464 @@
+      description: 'Sets which ARIA role will be used for the divider.',
+      table: {
+        category: 'Properties',
+        defaultValue: {
+          summary: 'presentation',
+        }
+      }
+    },
+    narrow_showDividers: {
+      control: {
+        type: 'boolean'
+      },
+      table: {
+        category: 'Narrow viewport properties',
+        defaultValue: {
+          summary: 'inherit'
+        }
+      },
+    },
+
+    // Children
+    children: {
+      description: 'A slot for children elements.',
+      table: {
+        category: 'HTML'
+      }
+    }
+  },
+};
+
+export const StackTemplate = ({
+  _debug,
+  _height,
+  _width,
+  direction,
+  gap,
+  gap_custom,
+  align,
+  alignWrap,
+  spread,
+  wrap,
+  showDividers,
+  dividerAriaRole,
+
+  narrow_direction,
+  narrow_gap,
+  narrow_gap_custom,
+  narrow_align,
+  narrow_alignWrap,
+  narrow_spread,
+  narrow_wrap,
+  narrow_showDividers,
+
+  children
+}) => {
+
+  let custom_styles  = {};
+
+  // Default values
+  direction = direction ?? 'block';
+  gap = gap ?? 'normal';
+  alignWrap = alignWrap ?? 'start';
+  dividerAriaRole = dividerAriaRole ?? 'presentation';
+
+  // Default narrow values
+  narrow_direction = narrow_direction ?? 'inherit';
+  narrow_gap = narrow_gap ?? 'inherit';
+  narrow_align = narrow_align ?? 'inherit';
+  narrow_alignWrap = narrow_alignWrap ?? 'inherit';
+  narrow_spread = narrow_spread ?? 'inherit';
+  narrow_wrap = narrow_wrap ?? 'inherit';
+  narrow_showDividers = narrow_showDividers ?? 'inherit';
+
+  // Gap
+  if (gap === 'custom') {
+    custom_styles['--Stack-gap'] = gap_custom;
+  }
+  if (narrow_gap === 'custom') {
+    custom_styles['--Stack-gap-whenNarrow'] = narrow_gap_custom;
+  }
+
+  // Null value for states that don't require a modifier class
+  align = align === 'stretch' ? null : align;
+  alignWrap = alignWrap === 'start' ? null : alignWrap;
+  spread = spread === 'start' ? null : spread;
+  wrap = wrap === 'nowrap' ? null : wrap;
+  gap = gap === 'normal' ? null : gap;
+
+  // Null value for inherit responsive values
+  narrow_direction = narrow_direction === 'inherit' ? direction : narrow_direction;
+  narrow_gap = narrow_gap === 'inherit' ? gap : narrow_gap;
+  narrow_align = narrow_align === 'inherit' ? align : narrow_align;
+  narrow_alignWrap = narrow_alignWrap === 'inherit' ? alignWrap : narrow_alignWrap;
+  narrow_spread = narrow_spread === 'inherit' ? spread : narrow_spread;
+  narrow_wrap = narrow_wrap === 'inherit' ? wrap : narrow_wrap;
+  narrow_showDividers = narrow_showDividers === 'inherit' ? showDividers : narrow_showDividers;
+
+  // Dividers logic
+  showDividers = wrap === 'wrap' ? false : showDividers;
+  narrow_showDividers = narrow_wrap === 'wrap' ? false : narrow_showDividers;
+
+  const hasDividers = showDividers || narrow_showDividers;
+
+  return (
+    <>
+      <div
+        className={clsx(
+          'Stack',
+          direction && 'Stack--dir-' + `${direction}-whenRegular`,
```

Comment:
Non-blocking nitpick: You don't have to use the old-school `+` concatenator - you could just make the whole think a string template.

```suggestion
          direction && `Stack--dir-${direction}-whenRegular`,
```